tracpro/polls/tasks.py

```diff
@@ -130,3 +131,35 @@ def org_task(self, org):
         are no longer on the remote.
         """
         apps.get_model('polls', 'Poll').objects.sync(org)
+
+
+@task
+def sync_questions_categories(org, polls):
+    # Create new or update SELECTED Polls to match RapidPro data.
+    from tracpro.polls.models import Poll, Question
+
+    # Save the associated Questions for this poll here
+    # now that these polls have been activated for the Org
+    selected_poll_names = [poll.name for poll in polls]
+
+    temba_polls = org.get_temba_client().get_flows(archived=False)
+    temba_polls = {p.uuid: p for p in temba_polls}
```

Comment:
I don't think we need this dictionary any more. (We're just extracting its `.values()`.)